package nail_distance_msgs.msgs;


message NailDistance
{

    required bool distance = 1;
  
}
